arch/cortex-m/src/lib.rs

```diff
@@ -76,7 +76,7 @@ pub unsafe extern "C" fn svc_handler_arm_v7m() {
     // is something other than 0xfffffff9, then we are coming from an app which
     // has called a syscall.
     cmp lr, #0xfffffff9
-    bne to_kernel
+    bne 100f
```

Comment:
```suggestion
    bne 100f // to_kernel
```

🪦 useful names, but we should maybe put on both ends?

Reply:
I realized that eventually, but harder when the asm is already commented properly